### ISSUE TITLE
Alias import support

### ISSUE THREAD

**Issue by PaulVDT** [kisstkondoros/gutter-preview#21]
It would be great if you could support alias imports eg in my jsconfig.json i define alias imports to be able to use @myalias to import globally example: 

```
{
  "compilerOptions": {
    "baseUrl": "./src/",
    "paths": {
      "@infonerd-web/*": ["web"],
      "@infonerd-native/*": ["native"],
      "@infonerd/*": ["*"]
    }
  }
}
```
![image](https://user-images.githubusercontent.com/11353388/45995783-d0c68300-c099-11e8-901a-ac531dfad9d1.png)

**Comment by kisstkondoros** [comment]
Nice idea!

I think implementing this feature would be high effort and at the moment I have no time to invest into it.

Proper solution:
Technical steps + problems as I've checked:
- Typescript compiler API would need to be used - very expensive (mostly memory)
- From that the module resolution could be read (perhaps in advance) - (caching + invalidation?)
- Using that information the path could be resolved in the collectEntries method in the server.ts

Workaround solution:
- use something like https://github.com/mazhlekov/tsc-resolve/blob/4a0d728cea71a0731b5bdfc7bd9ac532b485d251/src/tsc-resolve.ts#L47 for path resolution

I hope you don't mind but I'm going to close this for now. Perhaps I'll get back to this later this year. 

P.S.:
Pull Requests are welcomed, as always ^^

**Comment by kisstkondoros** [comment]
The feature is available in version 0.17.1 :fireworks:

**Comment by kisstkondoros** [comment]
... and hopefully fixed in 0.17.2 :)
